packages/playground/website/src/components/start-error-modal/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button } from '@wordpress/components';
+import { LogModal } from '../log-modal';
+import { useState } from '@wordpress/element';
+
+import css from './style.module.css';
+
+export const localStorageKey = 'playground-start-error-dont-show-again';
+
+export function StartErrorModal() {
+	const [dontShowAgain, setDontShowAgain] = useState(
+		localStorage.getItem(localStorageKey) === 'true'
+	);
+
+	if (dontShowAgain) {
+		return null;
+	}
+
+	const dismiss = () => {
+		localStorage.setItem(localStorageKey, 'true');
+		setDontShowAgain(true);
+	};
+
+	const description = (
+		<>
+			<p>
+				An error occurred while starting Playground. Please read the
+				logs below to understand the issue. If there is Invalid
+				blueprint error, the error will include the step which caused
+				the issue. You can review your blueprint and{' '}
+				<a href="https://wordpress.github.io/wordpress-playground/blueprints-api/troubleshoot-and-debug-blueprints">
+					check out the documentation
+				</a>{' '}
```

Comment:
Let's use `target="_blank"`

Reply:
Rewrite proposal:

There was a problem starting Playground. To figure out what went wrong, please take a look at the error logs provided below. If you see an "Invalid blueprint error," the logs will point out the specific step causing the issue. You can then double-check your blueprint. For more help, you can also [visit our documentation](https://wordpress.github.io/wordpress-playground/blueprints-api/troubleshoot-and-debug-blueprints).

Reply:
Would the crash modal replace this one if the error is a fatal crash?

Reply:
> Would the crash modal replace this one if the error is a fatal crash?

Yes, after fixing [eab466f](https://github.com/WordPress/wordpress-playground/pull/1336/commits/eab466f23299bd5d29757e27ceb0a498a633d673). But it doesn't look good. It first shows the logger and after that the error report modal. I need to find a way to not show logs in this case.

Reply:
It took me a bit to get to a clean solution, but this should work now [2dfec6f](https://github.com/WordPress/wordpress-playground/pull/1336/commits/2dfec6f280daa6ca749a34a0411ec08809185c17)